packages/block-library/src/quote/edit.js

```diff
@@ -82,6 +82,8 @@ export default function QuoteEdit( {
 				{ ( ! RichText.isEmpty( citation ) || isSelected ) && (
 					<RichText
 						identifier="citation"
+						tagName="cite"
+						style={ { display: 'inline-block' } }
```

Comment:
I wonder what we could do here to automatically fix the display property.

Reply:
yes, I wondered too but maybe it's ok to be explicit because it does change the styling, it's just that in all these situations we're actually ok with "inline-block" instead of "inline".